sisyphus/job.py

```diff
@@ -651,9 +655,80 @@ def _sis_all_path_available(self):
                 return False
         return True
 
+    def _sis_get_file_stats(self):
+        """
+        Returns a triple for every file below `work` and `output`: path, modification time, size.
+
+        These stats are written to the `usage` files by the `LoggingThread`, and read by
+        Job._sis_get_expected_file_sizes.
+
+        """
+        stats = []
+        below_work = pathlib.Path(self._sis_path(gs.WORK_DIR)).rglob("*")
```

Comment:
I think this runs directly before switching back from the job work directory to the experiment directory. This will fail from the job work directory since self._sis_path returns a relative path starting from the experiment directory.